packages/dev-env/src/mock/index.ts

```diff
@@ -164,6 +165,61 @@ export async function generateMockSetup(env: DevEnv) {
     }
   }
 
+  // make some naught posts & label them
+  const file = Buffer.from(labeledImgB64, 'base64')
+  const uploadedImg = await bob.agent.api.com.atproto.repo.uploadBlob(file, {
+    encoding: 'image/png',
+  })
+  const labeledPost = await bob.agent.api.app.bsky.feed.post.create(
+    { repo: bob.did },
+    {
+      text: 'naughty post',
+      embed: {
+        $type: 'app.bsky.embed.images',
+        images: [
+          {
+            image: uploadedImg.data.blob,
+            alt: 'naughty naughty',
+          },
+        ],
+      },
+      createdAt: date.next().value,
+    },
+  )
+
+  const filteredPost = await bob.agent.api.app.bsky.feed.post.create(
+    { repo: bob.did },
+    {
+      text: 'reallly bad post should be deleted',
+      createdAt: date.next().value,
+    },
+  )
+
+  const ctx = env.listOfType(ServerType.PersonalDataServer)[0].ctx
+  if (ctx) {
+    await ctx.db.db
+      .insertInto('label')
+      .values([
+        {
+          sourceDid: ctx.cfg.labelerDid,
+          subjectUri: labeledPost.uri,
+          subjectCid: labeledPost.cid,
+          value: 'nudity',
+          negated: 0,
+          createdAt: new Date().toISOString(),
+        },
+        {
+          sourceDid: ctx.cfg.labelerDid,
+          subjectUri: filteredPost.uri,
+          subjectCid: filteredPost.cid,
+          value: 'csam',
```

Comment:
```suggestion
          value: 'dmca',
```

Reply:
hey @pfrazee i changed this one cause it felt like a weird label in our dev-env 🙃

Reply:
ah right good catch. i shouldve checked